thumbor/loaders/file_loader.py

```diff
@@ -39,18 +39,17 @@ def load(context, path, callback):
     if not exists(file_path):
         file_path = unquote(file_path)
 
-    if exists(file_path):
+    result.successful = exists(file_path)
```

Comment:
Since you are touching this, do you mind putting this entire thing in a try block and setting result.successful to False before the try block? That way if the file exists but we do not have access to it or anything happens we still get a sane result.

Reply:
Sure, asap.

Reply:
Sorry for the waiting, I'm have been busy these days. Modified it, used a try-except block.